tests/Hybrid/CreateTable/MiscCreateTest.php

```diff
@@ -14,7 +14,7 @@ class MiscCreateTest extends BaseTest
     /** @test */
     public function all_keys_are_added_in_create_columnstore()
     {
-        if (version_compare(Application::VERSION, '8.12.0', '<=')) {
+        if (version_compare(Application::VERSION, '8.79.0', '<=')) {
```

Comment:
I would prefer that you split the fulltext index out of this test into a separate test rather than reducing coverage on all the other index types.

Reply:
see comment below - let's just remove fulltext from this file

Reply:
100%